File: app/src/main/java/com/renj/fragment/MyApplication.java
package com.renj.fragment;

import android.app.Application;

/**
 * ======================================================================
 * <p>
 * 作者：Renj
 * <p>
 * 创建时间：2021-10-13   17:46
 * <p>
 * 描述：
 * <p>
 * 修订历史：
 * <p>
 * ======================================================================
 */
public class MyApplication extends Application {
    private static Application application;

    @Override
    public void onCreate() {
        super.onCreate();

        application = this;
    }

    public static Application getApplication() {
        return application;
    }
}

File: app/src/main/java/com/renj/fragment/MyApplication.kt
package com.renj.fragment

import android.app.Application

/**
 * ======================================================================
 *
 * 作者：Renj
 *
 * 创建时间：2021-10-13   17:46
 *
 * 描述：
 *
 * 修订历史：
 *
 * ======================================================================
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        application = this
    }

    companion object {
        lateinit var application: Application
            private set
    }
}